### ISSUE TITLE
Issue with Setting enable_secure_boot = "Off"

### ISSUE THREAD

**Issue by furrycoat21** [taliesins/terraform-provider-hyperv#41]
### Terraform Version
2021/02/11 12:24:24 [INFO] Terraform version: 0.14.6  
2021/02/11 12:24:24 [INFO] Go runtime version: go1.15.6
+ provider registry.terraform.io/taliesins/hyperv v1.0.1

### Affected Resource(s)
hyperv_machine_instance

### Terraform Configuration Files
```
resource "hyperv_machine_instance" "fedora_33" {
  vm_firmware {
    enable_secure_boot = "Off"
  }
}
```
### Debug Output
https://gist.github.com/furrycoat21/d4490cc11ee9527163ca048e3adfc747 < 1st Run 
https://gist.github.com/furrycoat21/09ab89f7f8b3b8d7919df12d89831de4 < 2nd Run

### Expected Behavior
VM Created with Secure Boot Off

### Actual Behavior
VM is created with Secure Boot ON, Then a second run sets Secure Boot to Off. 
Issue is that the VM cannot be auto Started on Deploy. You have to set VM state to Off, to have it successfully run the second time. 

### Steps to Reproduce
`terraform apply`

**Comment by furrycoat21** [comment]
So I think the issue (as I have found it) is that the VM is on when it tries to run the powershell in createOrUpdateVmFirmwareTemplate. Since its a runFireAndForgetScript it never gets back an error where the VM needs to be Off to Set it. 

Adding a line in the createOrUpdateVmFirmwareTemplate powershell to turn off the VM before the Firmware gets set fixes it.

**Comment by taliesins** [comment]
I have updated to all the latest packages in #44 

When running a firmware change on a running machine it does indeed fail. What should be the correct behaviour? Do we turn the machine off and on or do we fail. Do we turn it off at the beginning and then on at the end of all vm instance related changes, or do we do it per a step?

This is the output I receive:
```
Error: run command operation returned code=1
stderr:

stdOut:
Set-VMFirmware : 'web_server_g2' failed to modify settings.
Cannot modify the secure boot property.
'web_server_g2' failed to modify settings. (Virtual machine ID 892F3D2F-4469-46E3-8DE8-FDCF39FC6B2E)
Cannot modify the secure boot property while the virtual machine is running.
At C:\Users\taliesins\AppData\Local\Temp\shell-terraform-1pL3BXEywANj48A3gLZeF6yfIX0.ps1:15 char:1
+ Set-VMFirmware @SetVMFirmwareArgs
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Set-VMFirmware], VirtualizationException
    + FullyQualifiedErrorId : OperationFailed,Microsoft.HyperV.PowerShell.Commands.SetVMFirmware

  on main.tf line 60, in resource "hyperv_machine_instance" "web_server_g2":
  60: resource "hyperv_machine_instance" "web_server_g2" {
```

**Comment by taliesins** [comment]
Implemented this functionality in #45
